src/services/devHandler/newsProcessDevProcess.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable camelcase */
+const {v4: uuidv4} = require('uuid');
+const {newsQueue} = require('../../config');
+const {checkIfValidURL, successResponse, errorResponse} = require('../../utils');
+
+const newsProcessDevProcess = async (req, res) => {
+  const bodyData = req.body?.message;
+  const id_feed = req.body?.id || false;
+  const user_id = req.body?.actor?.id || false;
+  const topics = req.body?.topics || [];
+  const duration_feed = req.body?.duration_feed || '';
+
+  const options = {
+    jobId: uuidv4(),
+    removeOnComplete: true
+  };
+
+  console.log('check if valid url', checkIfValidURL(bodyData));
+
+  if (checkIfValidURL(bodyData)) {
+    const getJob = await newsQueue.add(
+      {
+        body: checkIfValidURL(bodyData),
+        id_feed,
+        user_id,
+        topics,
+        duration_feed
+      },
+      options
+    );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `checkIfValidURL` function is called twice - once for logging and once for the actual check. This could potentially be an expensive operation, so it's better to call it once, store the result in a variable, and then use that variable.

```diff
  const isValidUrl = checkIfValidURL(bodyData);

  if (isValidUrl) {
    const getJob = await newsQueue.add(
      {
        body: isValidUrl,
        id_feed,
        user_id,
        topics,
        duration_feed
      },
      options
    );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->